nf_core/list.py

```diff
@@ -198,6 +201,10 @@ def filtered_workflows(self):
             else:
                 # We didn't hit a break, so all keywords were found
                 filtered_workflows.append(wf)
+
+        # remove archived worflows; show_archived is False by default
+        if not self.show_archived:
+            filtered_workflows = [wf for wf in filtered_workflows if wf.archived == False]
```

Comment:
This also overwrites `filtered_workflows` again, so will essentially break searching by keyword